src/plugins/data/public/ui/index.ts

```diff
@@ -21,3 +21,5 @@ export { SuggestionsComponent } from './typeahead/suggestions_component';
 export { IndexPatternSelect } from './index_pattern_select';
 export { FilterBar } from './filter_bar';
 export { applyFiltersPopover } from './apply_filters';
+// temp export
```

Comment:
Why is this a temp export?

Reply:
++ Yeah wondering the same thing

Reply:
Because once the entire `QueryStringInput` component is moved into NP, `QueryLanguageSwitcher` becomes its implementation detail.

So the export is there ATM is due to a partial migration.